pkg/docker/containers-view.jsx

```diff
@@ -185,14 +209,59 @@ var ContainerList = React.createClass({
         this.setState({ containers: containers });
     },
 
+    setNewProblem: function (c_id, url, message) {
+        /* New problem is always displayed, no matter if the same problem is
+         * already shown. It is because user may be interested into dynamic
+         * watching of the problems occurring. After refreshing the site, only
+         * the latest occurrence is displayed.
+         */
+        var known_problems = this.state.problems;
+        if ( c_id in known_problems )
+            known_problems[c_id].push([url, message]);
+        else
+            known_problems[c_id] = [[url, message]];
+        this.setState({ problems: known_problems });
+    },
+
+    newProblemOccurred: function (event, problem_path) {
+        var entry = this.problems_client.proxy('org.freedesktop.Problems2.Entry', problem_path);
+        var self = this;
+        entry.wait(function() {
+            util.find_container_log(entry, self.setNewProblem);
+         });
+    },
+
     componentDidMount: function () {
+        var self = this;
+        this.problems_client = cockpit.dbus('org.freedesktop.problems', { superuser: "try" });
+        this.service = this.problems_client.proxy('org.freedesktop.Problems2', '/org/freedesktop/Problems2');
+        this.problems = this.problems_client.proxies('org.freedesktop.Problems2.Entry', '/org/freedesktop/Problems2/Entry');
+        this.problems.wait(function() {
+            try{
+                self.service.GetSession().done(function(session_path) {
+                    var session = self.problems_client.proxy('org.freedesktop.Problems2.Session', session_path);
+                    session.wait(function() {
+                        session.Authorize({}); //ignore return code
+                    });
+                });
+            }
+            catch (err) {
+                // no ABRT present, continue
+            }
+        });
+
         $(this.props.client).on('container.containers', this.containersChanged);
         $(this.props.client).on('container.container-details', this.containersChanged);
+        $(this.service).on("Crash", this.newProblemOccurred);
```

Comment:
Argh jQuery in React. Not new in this file, so cleaning up the existing ones is a thing for a separate PR. But it would be nice to not add new jQuery calls here. Can you use `addEventListener()` instead?